src/components/RandomLoanSelector/randomLoanSelector.vue

```diff
@@ -2,7 +2,7 @@
 	<transition name="kvfade">
 		<div
 			class="section-wrapper random-loan-cards">
-			<div class="section-container">
+			<div v-if="randomLoans.length" class="section-container">
```

Comment:
don't show loan carousel until there are loans, fix the initial view of an empty interface with the arrows misaligned

Reply:
🧠